### DISCUSSION TITLE
Printer 0,0 position

### DISCUSSION THREAD

**Discussion by dfox1787** [supermerill/SuperSlicer#1695] [Q&A]
Hi 

My printer start position is top right and i cant seem to change this in super slicer. As a result the printer prints the part in the wrong location. 

used s3d for 7 years now but fancied using something with more features but cant seem to match the settings.

**Comment by supermerill** [comment]
It shouldn't change anything if it's top right vs bottom left. 
If it was top left, that may be a bit more awkward as the object may print mirrored.

**Reply by dfox1787** [reply]
> It shouldn't change anything if it's top right vs bottom left. If it was top left, that may be a bit more awkward as the object may print mirrored.

It does seem to make a difference or there is something different in super slicer. If I load the same file in S3d it prints in the center as it should, In super slicer it prints on the right.

**Reply by supermerill** [reply]
Do you have a picture or something?
I don't see why. Maybe the bed size is wrong?

**Reply by dfox1787** [reply]
> Do you have a picture or something? I don't see why. Maybe the bed size is wrong?
Ive just sat and watched it. Whats happening is the hot end is traveling at the rear of the printer and hitting the bed clip causing it to be offset. 

How do we start at the front on the printer? S3D primes on the right hand side then moves to the middle. 
![Capture](https://user-images.githubusercontent.com/28814095/138695384-8c246551-c58e-465b-b59a-6f5d8b94b0af.PNG)

**Reply by supermerill** [reply]
Depends on your start gcode.
If you put a G1 X200,Y200 at the end of it it should "start" at 200:200

for the bed clip I'm working on something to avoid travels hitting it.

**Reply by dfox1787** [reply]
> Depends on your start gcode. If you put a G1 X200,Y200 at the end of it it should "start" at 200:200
> 
> for the bed clip I'm working on something to avoid travels hitting it.

sorted thank you. i copied my start and end scripts from s3d
